packages/transformer/src/test/standalone/ECReferenceTypesCache.test.ts

```diff
@@ -185,6 +190,20 @@ describe("ECReferenceTypesCache", () => {
     ).to.equal(ConcreteEntityTypes.Element);
   });
 
+  it("should handle QueryView", async () => {
+    if (!Semver.gte(iTwinCoreBackendVersion, "4.6.0")) {
+      return; // Pre 4.6.0 does not have QueryView support. https://www.itwinjs.org/bis/domains/ecdbmap.ecschema/#queryview
+    }
+    const thisTestRefCache = new ECReferenceTypesCache();
+    const ecdbMapVersion =
+      emptyWithBrandNewBiscore.querySchemaVersion("ECdbMap");
+    assert(ecdbMapVersion !== undefined);
+    assert(Semver.gte(ecdbMapVersion, "2.0.4"));
+    await emptyWithBrandNewBiscore.importSchemas([testSchemaPathWithQueryView]);
+    emptyWithBrandNewBiscore.saveChanges();
+    await thisTestRefCache.initAllSchemasInIModel(emptyWithBrandNewBiscore);
```

Comment:
How do we know if this test were to fail? Do we just expect initAllSchemasInIModel() to always produce an error if it failed? Is there anything on thisTestRefCache that we can assert to prove the changes from relInfoFromRelClass worked?